system/include/emscripten/wasmfs_internal/backend.h

```diff
@@ -4,10 +4,11 @@
 // found in the LICENSE file.
 
 // This file defines the modular backend abstract class.
+// The declarations from this files should only be used by file system implementers
 
 #pragma once
 
-#include "file.h"
+#include <emscripten/wasmfs_internal/file.h>
```

Comment:
Can you leave these as-is, if the files remain in the same directory?

Reply:
I will revert for headers..., but it may take until next weekend...

Reply:
Ok, fixed.